### PR TITLE
Python3 compatibility, bugfixes and more

### LINKED ISSUE #52: confirmation-mail: no link for deactivation
I just received a confirmation-mail for a freshly registered address:
```
Hey,

du hast folgende IPs bekommen:

Mesh-IPs:
 * a.f.f.e/27


Wenn du die IP-Adressen nicht mehr brauchst kannst du sie folgendem Link
löschen:



Grüße
Freifunk Berlin
```

This mail is missing the link to return the addresses, which should be printed after "folgendem Link
löschen:"

### PR DESCRIPTION
**Pull request by rtznprmpftl**
This is a fairly big commit, that contains more than it should.

- Migrated everything to python3
  - ran 2to3 to automatically convert the code
  - updated dependencies to their latest versions
  - smaller fixes to the code where neccessary

- cleaned up the  code a bit (e.g. unused imports, code formatting etc)

- fixed the language a bit (https://de.wikipedia.org/wiki/Email != https://de.wikipedia.org/wiki/E-Mail)
- fixed #52

### CODE REVIEW

File: scripts/legacy_importer/__init__.py [hunk 1]
```diff
@@ -17,22 +18,23 @@ def legacy_import(db_user, db_pass, db_host, db_name):
...
+                    'tags': ['legacy', 'imported']
                 }
                 get_api().create_prefix_by_cidr(c, user[3], pool, data)
 
-                print("\t* %s saved." % c)
+                print(("\t* %s saved." % c))
             except NipapDuplicateError:
-                print("\t* %s already exists." % c)
+                print(("\t* %s already exists." % c))
+
```
Comment (Akira25) [inline scripts/legacy_importer/__init__.py:37]: Do we really need two ``(())`` brackets nested for the print-statements here?

File: app/templates/wizard/form.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
           <form class="form-horizontal col-md-8" role="form" action="{{url_for('.wizard_form', router_id = "/".join(router.id)) }}" method="post">
 
             {{ form.csrf_token }}
-            {{ render_bootstrap_field(form.hostname, "Spitzname des Routers", "Ein kurzer und eindeutiger Name für deinen Router (z.B. <em>funkenpumpe</em> oder <em>HermannStr42_VH</em>). Dieser Name wird nur intern für dich hinterlegt.") }}
+            {{ render_bootstrap_field(form.hostname, "Spitzname des Routers", "Ein kurzer und eindeutiger Name für deinen Router (z.B. <em>funkenpumpe</em> oder <em>HermannStr42_VH</em>). Dieser Name wird nur intern für dich hinterlegt und muss zwischen 4 und 32 Zeichen lang sein.") }}
```
Comment (Akira25) [inline app/templates/wizard/form.html:15]: Good Idea might be to add, that this name should match the name, which gets typed into the firmware-wizard. That was a thing that @noki mentioned some time ago. Once we got the new firmware-wizard, we can set more strong incentives for this via the config-json-upload.

File: app/templates/index.html [hunk 3]
```diff
@@ -39,7 +38,7 @@
         <div class="col-md-12 text-center">
           <iframe src="//player.vimeo.com/video/64814620?portrait=0&title=0&byline=0"
                   width="500" height="281" frameborder="0"
-                  webkitallowfullscreen mozallowfullscreen allowfullscreen>
+                  webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen>
```
Comment (Akira25) [inline app/templates/index.html:41]: I don't know exactly, what these attributes do. But maybe you missed `allowfullscreen=true`?

File: app/templates/index.html [hunk 4]
```diff
@@ -10,10 +10,9 @@
           Du bist hier durch Zufall oder mit Absicht auf der Adressenvergabe
           der <em><a href="https://berlin.freifunk.net/">Berliner
               Freifunker_innen</a></em>
-          gelandet. Falls du Hilfe brauchst bei der Einrichtung deines Freifunk-
-          Routers folge am besten der
-          <em><a href="https://wiki.freifunk.net/Berlin:Firmware:Howto">Anleitung</a></em>
-          .
+          gelandet. Falls du Hilfe bei der Einrichtung deines Freifunk-
+          Routers brauchst, folge am besten der
+          <em><a href="https://wiki.freifunk.net/Berlin:Firmware:Howto">Anleitung</a></em>.
```
Comment (Akira25) [inline app/templates/index.html:15]: This link changed to `https://wiki.freifunk.net/Berlin:Firmware/HowTo`

**Review by Akira25 — COMMENTED** [review]
In overall this looks good to me. Did you do some runtime-testing on the development-machine?

File: scripts/legacy_importer/__init__.py (hunk 1 above)
Comment (rtznprmpftl) [reply scripts/legacy_importer/__init__.py:37]: i don't know.

It was done by 2to3 https://docs.python.org/3/library/2to3.html#to3fixer-print

but i will take a look at this

File: app/templates/index.html (hunk 3 above)
Comment (rtznprmpftl) [reply app/templates/index.html:41]: this was maked as an issue by a linter, but judging by: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen i mgiht just replace it with the newer attribute

File: app/templates/index.html (hunk 4 above)
Comment (rtznprmpftl) [reply app/templates/index.html]: fixed

File: app/__init__.py [hunk 5]
```diff
@@ -3,11 +3,11 @@
 from flask import Flask, render_template
 from .wizard import wizard
 from .exts import db, mail, migrate
-from .wizard import wizard
```
[the remark this answers is not in the document]
Comment (rtznprmpftl) [reply app/__init__.py:6]: because it was a duplicate (look 2 lines up)

[the remark this answers is not in the document]
Comment (Akira25) [reply app/__init__.py:6]: oh. Haha. Nevermind.

**Pull request merged**